src/Mnemonic.h

```diff
@@ -12,6 +12,10 @@ namespace TW {
 
 /// BIP39 Mnemonic Sentence handling.
 class Mnemonic {
+public:
+    static constexpr size_t MaxWords = 24;
```

Comment:
Why 24?

Reply:
Trezor lib supports 12 and 24 words (and a few in between), and it has some implicit limits.  I thinks the standard is also does not allow more.